app/main.py

```diff
@@ -2,0 +1,24 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    # Check each friend using try/except
+    for friend in friends:
+        try:
+            # Try to allow the friend to visit the cafe
+            cafe.visit_cafe(friend)
+        except VaccineError:  # Removed 'as ve'
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
```

Comment:
The `NotWearingMaskError` exception should also have a descriptive error message. It's mentioned in the checklist. You can do this in the `NotWearingMaskError` class.